### PR TITLE
bug fix for issue number #912

### PR DESCRIPTION
**Pull request by biplavmz**
bug issue solve for bug number #912

### CODE REVIEW

File: src/components/Post/PostHeader.jsx [hunk 1]
```diff
@@ -136,6 +137,10 @@ const PostHeader = ({ postId, user, postData, postHasImages, timestamp }) => {
         <p className="post__time">{time}</p>
       </Link>
       <div className="social__icon__last">
+       
+        {
+          
+          (user?.uid == username || location.pathname == "/dummygram/" || location.pathname == "/dummygram/favourites" || location.pathname == "/dummygram/search") &&
```
Comment (narayan954) [inline src/components/Post/PostHeader.jsx:143]: you still didn't follow what I asked in my review. I asked to add this condition to only hide visit profile option and any relevant option.

**Review by narayan954 — CHANGES_REQUESTED** [review]

**Comment by narayan954** [comment]
Also you created separate pr  instead of fixing that pr(#928), dont do that, I won't be checking every new pr for the same issue, and close older one. Don't create duplicates. You can just add commit to the pr and it'd get reflected here. 
If you create another duplicate pr for this same issue instead of working on this only, I'll void this submission and not allow you to contribute in this repository. You've doing this for quite some time now.

**Comment by biplavmz** [comment]
ok @narayan954

File: src/components/Post/PostHeader.jsx (hunk 1 above)
Comment (biplavmz) [reply src/components/Post/PostHeader.jsx:143]: @narayan954  bro i think it working fine 
image

image

Comment (biplavmz) [reply src/components/Post/PostHeader.jsx:143]: images from search pages and profile pages
in profile pages there is no other option on view profile is there that's why hidding the three dot's

Comment (narayan954) [reply src/components/Post/PostHeader.jsx:143]: > @narayan954 bro i think it working fine image
> 
> image

it's on profile page bro, I think you're showing saved page broooo

Comment (narayan954) [reply src/components/Post/PostHeader.jsx:143]: > images from search pages and profile pages in profile pages there is no other option on view profile is there that's why hidding the three dot's

there's these options in owner's profile
image
and yes on other's profile, there're no other option than that, but we just need to remove this option, so it works globally. 
image

Comment (narayan954) [reply src/components/Post/PostHeader.jsx:143]: I think maybe remove this ... at once as you're saying would be better to remove the complications in a single move, so I'm allowing your changes... sorry for the inconvenience

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**